### ISSUE TITLE
two cursors even when putting the actual path for the cursor

### ISSUE THREAD

**Issue by beh-10257** [ValveSoftware/gamescope#886]
basically while using gamescope I have two cursors and gamescope will not use my actual theme
I am on Wayland Hyprland
even after putting a cursor path image same issue there are two cursors
![image](https://github.com/ValveSoftware/gamescope/assets/77834479/2063bf24-a7df-4335-9d28-75caae829c91)
with annotations
![image](https://github.com/ValveSoftware/gamescope/assets/77834479/6d2d59d7-ab01-4d09-b21a-320bf991a338)

basically the expected behaviors is to gamescope to actually use my theme (sometimes it uses X cross but yeah)
and the other expected behavior is to just show one cursor

**Comment by Meekology** [comment]
Gamescope version: 3.12.0-1
Kernel: 6.4.6-zen1-1-zen
Driver: Nvidia-dkms 535.86.05-2

I also have this issue. No matter the cursor arguments passed to gamescope it'll draw the "real" cursor in game as a small garbled mess like the photos above. I cannot seem to fix it with the --cursor or --force-grab-cursor arguments. One thing to note is that for whatever reason the more "aggressive" the up-scaling the more the "real" cursor is apparent. 

Weirdly enough in version 3.11 I had the issue as well though only on some instances. I use gamescope with PrismLauncher (A minecraft launcher for managing instances). It was very apparent in some instances, very mild in others, and completely fixxed in some. I have been trying to find a solution to this for days to no avail.

**Comment by Meekology** [comment]
Alright, small update. I removed "force grab cursor" and while I think this fixxed it (bit hard to tell, as I'll explain in a moment) caused a major issue -- My mouse seems to adjust the camera on one plane of one axis at an _**extremely**_ high sensitivity. This results in the camera facing down towards the floor and any mouse movement causing it to spin excessively quickly to the right. I cannot turn any other direction no matter the movement of the mouse. Only to the right. 

Theory? I think that for some reason gamescope thinks my mouse is a controller joystick input or something? Only thing I can imagine that would cause this behavior.

**Comment by Meekology** [comment]
No matter what this is still an issue for me. Sometimes I won't have it for no describable reason and other times It'll be there (95% of the time). It's extremely frustrating to deal with.
